README.md

```diff
@@ -36,22 +36,28 @@ NOTE: The default configuration will build 64-bit binaries for maximum security
 **IMPORTANT**: Currently, the `MAX_PATH` path length restriction (which is 260 characters by default) must be lifted in for our Python build scripts. This can be lifted in Windows 10 (v1607 or newer) with the official installer for Python 3.6 or newer (you will see a button at the end of installation to do this). See [Issue #345](https://github.com/Eloston/ungoogled-chromium/issues/345) for other methods for other Windows versions.
 
 1. Setup the following:
-
-    * 7-zip
-    * Python 3.6 - 3.9 or 3.10.2+ (for build and packaging scripts used below)
-        * At the end of the Python installer, click the button to lift the `MAX_PATH` length restriction.
-        * Check that your `PATH` does not contain the `python3` wrapper shipped by Windows, as it will only promt you to install Python from the Microsoft Store and exit. See [this question on stackoverflow.com](https://stackoverflow.com/questions/57485491/python-python3-executes-in-command-prompt-but-does-not-run-correctly)
+    * 7-Zip
+    * Python 3.6 - 3.9 or 3.10.2+ (for build and packaging scripts used below); Python 3.11 and above is not supported.
+    * If Python is not installed from the Microsoft Store:
+    	* Check "Add python.exe to PATH" before install.	
+        * Click the button to lift the `MAX_PATH` length restriction at the end of the Python installer.
+        * Check that your `PATH` does not contain the `python3` wrapper shipped by Windows, as it will only prompt you to install Python from the Microsoft Store and exit. See [this question on stackoverflow.com](https://stackoverflow.com/questions/57485491/python-python3-executes-in-command-prompt-but-does-not-run-correctly)
```

Comment:
I think this is not required anymore, the python installer adds the installation path to the beginning of the PATH, therefore the python executable from your installation is used.

Reply:
Python doesn't add the install path to PATH by default, but instead installs the Python launcher into the Windows folder (which is in PATH).

Reply:
When `Add python to PATH` is checked in the installer (as it is described above), python will be added to the beginning of the PATH variable. Even if there is an other python in the PATH, the newly installed python will be used.

Reply:
Could you please re-add the line `Click the button to lift the `MAX_PATH` length restriction at the end of the Python installer.`. Even though it is menitioned below, it should be listed there as well.